services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaServiceITest.kt

```diff
@@ -173,11 +173,11 @@ class HankeKayttajaServiceITest : DatabaseTest() {
         fun `When no kayttaja should return null`() {
             val hankeWithApplications = hankeFactory.saveGenerated(userId = USERNAME)
             val hankeId = hankeWithApplications.hanke.id!!
-            val jou = hankeKayttajaService.getKayttajaByUserId(hankeId)!!
+            val jou = hankeKayttajaService.getKayttajaByUserId(hankeId, USERNAME)!!
```

Comment:
What's jou?

Reply:
It is my go-to placeholder for variables I don't know how to name. This is the nasty side-effect. Fixed.